src/azul/lambdas.py

```diff
@@ -97,11 +97,16 @@ class Lambdas:
     def _lambda(self):
         return aws.lambda_
 
-    def list_lambdas(self) -> list[Lambda]:
+    def list_lambdas(self, deployment: str | None = None) -> list[Lambda]:
+        def lambda_deployment(function: JSON) -> str | None:
```

Comment:
Can this really return `None`?

"deployment-specific" in commit title, not "deployment specific"

Reply:
> Can this really return `None`?

No, the property is `None` in the base `environment.py`, but will never be `None` in practice since it is overwritten by the deployment's `environment.py`. Fixed in fixup commit.